### DISCUSSION TITLE
Make default browser

### DISCUSSION THREAD

**Discussion by vmt1997** [Floorp-Projects/Floorp#953] [Feature Request]
Portable version of Floorp don't have a features on setting page to make default browser

**Comment by creeper-0910** [comment]
The portable version is not intended to be set as the default browser.

**Reply by vmt1997** [reply]
I use scoop.sh as a package manager and is always install app as portable versions, so I can't switch floorp to default like in firefox download form scoop, I think is should be nine to have that features
